.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,28 @@
+### Prerequisites
+
+- [ ] I have written a descriptive issue title
+- [ ] I have verified that I am running the latest version of Nancy
+- [ ] I have verified if the problem exist in both `DEBUG` and `RELEASE` mode
+- [ ] I have searched [open issues](https://github.com/NancyFx/Nancy/issues) to ensure it has not already been reported

```

Comment:
Fair point

On Monday, 4 April 2016, Brian Surowiec <notifications@github.com> wrote:

> In .github/ISSUE_TEMPLATE.md
> <https://github.com/NancyFx/Nancy/pull/2393#discussion_r58451100>:
>
> > @@ -0,0 +1,28 @@
> > +### Prerequisites
> > +
> > +- [ ] I have written a descriptive issue title
> > +- [ ] I have verified that I am running the latest version of Nancy
> > +- [ ] I have verified if the problem exist in both `DEBUG` and `RELEASE` mode
> > +- [ ] I have searched [open issues](https://github.com/NancyFx/Nancy/issues) to ensure it has not already been reported
>
> Would it be better to say open and closed issues? I saw the angular-ui
> team
> <https://github.com/angular-ui/bootstrap/blob/master/.github/ISSUE_TEMPLATE.md>
> mentioned both since the issue could have been resolved already.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/NancyFx/Nancy/pull/2393/files/abe41fa4cc5b190deadb9300845e76df2d06ab10#r58451100>
>